src/gtk/textctrl.cpp

```diff
@@ -800,6 +800,14 @@ bool wxTextCtrl::Create( wxWindow *parent,
 
     if (multi_line)
     {
+        // By default, accepts-tab property is true, corresponding to the
+        // expected behaviour when wxTE_PROCESS_TAB is specified, so reset it
+        // if necessary.
+        if ( !(style & wxTE_PROCESS_TAB) )
+        {
+            gtk_text_view_set_accepts_tab(GTK_TEXT_VIEW(m_text), FALSE);
```

Comment:
It would be simpler to just always make the call:
`gtk_text_view_set_accepts_tab(GTK_TEXT_VIEW(m_text), (style & wxTE_PROCESS_TAB) != 0);`
Less code, and it won't do anything if the value isn't changing, plus no dependence on what the default is.